calculator_submit.rb

```diff
@@ -0,0 +1,58 @@
+# this version is an experiment with while loops and the .include method
+puts "Hi! Let's do math!"
+
+puts "Please tell me which kind of math you would like to do"
+math_type = gets.chomp.downcase
+
+#this array includes all valid math types
+#the while loop loops through the array with the .include method with the
+#argument of math_type as the paramenter
+#as long as the array.include?(math_type) is false the invalid input message
+#will print to the screen
+valid_math_types = ['+', 'add', 'addition', '-', 'sub', 'subtraction', '*',
+  'mult', 'multiply', 'multiplication', '/', 'div', 'divide', 'division', '^',
+  'exponent', 'exponential', '%', 'modulo', 'mod']
+  while valid_math_types.include?(math_type) == false
+    puts "Please insert a valid math operation"
+    math_type = gets.chomp.downcase
+  end
+
+puts "Great! Please enter the first number for #{math_type}."
+first_number = gets.chomp.to_i
+#Consider checking for invalid string input THEN changing to an integer
+
+puts "And the second number..."
+second_number = gets.chomp.to_i
+#Consider checking for invalid string input THEN changing to an integer
+
+  if
```

Comment:
The if statement here is nice and readable, but it might be worth looking into using ruby's cool `case` statement feature. (http://www.skorks.com/2009/08/how-a-ruby-case-statement-works-and-what-you-can-do-with-it/)

To be fair, I've only used this rarely (and don't see people use it a lot), I just think this is a nice application for it.